lametro/management/commands/generate_tag_analytics.py

```diff
@@ -20,12 +22,27 @@ class Command(BaseCommand):
     help = "This command produces a CSV file that lists each Board Report's tags and \
         uploads it to the 'LA Metro Reports' folder in Google Drive."
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-l',
+            '--local',
+            action='store_true',
+            help='Output the generated CSV file to the current directory'
+        )
+
     def handle(self, *args, **options):
+        local = options['local']
 
         csv_string = self.generate_tag_analytics()
         date = datetime.today().strftime('%m_%d_%y')
         output_file_name = f'{date}_tag_analytics.csv'
 
+        if local:
+            with open(output_file_name, 'wb') as f:
+                copyfileobj(csv_string, f)
+
+            return
```

Comment:
Can you say more about what's going on here?

Reply:
Sure, this is a way to write the StringIO object containing the CSV to a local file. It's already a file-like object so you have to use `shutil.copyfileobj`.

I included the `-l` flag to make local testing easier.